### ISSUE TITLE
Server DBC locales 

### ISSUE THREAD

**Issue by gmodmania** [callmephil/SunWellCore#3]
Change DBC.Locale = 255 to 2 nothing happend.

Server log:
Using enUS DBC Locale as default. All available DBC locales: frFR enUS

Core problem or what?

**Comment by callmephil** [comment]
Yes, Core Doesn't handle different locales languages atm. i'm working on it. it's seems that it came later on TrinityCore.

**Comment by gmodmania** [comment]
Ok, i will try to fix this too

**Comment by gmodmania** [comment]
Any progress?

**Comment by callmephil** [comment]
found some related information but got a huge list of todo things so i didn't get the time to take a look properly. looking at all commit done by trinity to find out implementation of "Locale"

**Comment by Devalenok** [comment]
This is strange even by 52 Reva not suitable DBC

**Comment by callmephil** [comment]
i'm currently on it. i have to implement & update locales manually for each locales. actual working locales for items. doing it for the others and pushing as soon as everything works.

**Comment by gmodmania** [comment]
Will you update it soon?

**Comment by callmephil** [comment]
i'm getting a lot of todo right now i didin't work on it since last friday. will be more free next week.

**Comment by tea91** [comment]
Yea! Big fighting for callmephil! :+1: 

Plz check to KoKR too.

**Comment by gmodmania** [comment]
Any news about locales? So wating for this update!

**Comment by Motorheadx** [comment]
how things are going to locales?

**Comment by tea91** [comment]
Still waiting :+1:

**Comment by gmodmania** [comment]
Can you give as hope to wait? :)

**Comment by callmephil** [comment]
working on it takes a lot of time since 20 or 30 % of the functions used have changed/not present.

**Comment by DevQuad** [comment]
[Please check, works well for me.](https://gist.github.com/DevQuad/aa8fdcd0d7240fcf88fc4f495a9a0a9e)

**Comment by DevQuad** [comment]
Сheck [this](https://gist.github.com/DevQuad/aa8fdcd0d7240fcf88fc4f495a9a0a9e).

**Comment by FALL1N1** [comment]
so many ungrateful people here.. lol

**Comment by FALL1N1** [comment]
easy fix, hint: http://prntscr.com/bhnmvc

**Comment by tea91** [comment]
https://github.com/JunkyBulgaria // nah.. plz uploading .diff
